File: 3-2/Yazılım Mühendisliği/projeler/2022/onder_gormez/Person.cpp
#include "Person.h"

namespace LanguageSchool
{
    class Person::Person(std::string name, std::string surname, std::string mobilephone, std::string phone, std::string email, int age, std::string gender, std::string address) : m_name(name),
                                                                                                                                                                                   m_surname(surname),
                                                                                                                                                                                   m_mobilephone(mobilephone),
                                                                                                                                                                                   m_phone(phone),
                                                                                                                                                                                   m_email(email),
                                                                                                                                                                                   m_age(age),
                                                                                                                                                                                   m_gender(gender),
                                                                                                                                                                                   m_address(address)

    {
    }

    void Person::setName(std::string newName)
    {
        m_name = newName;
    }

    std::string Person::getName()
    {
        return m_name;
    }
} // LanguageSchool

File: 2-1/Nesneye Yonelik Programlama/kodlar_lablar/kodlar/2024/batuhan_odcikin/Java/denemedeneme/Person.java
package paket1;

public class Person {
	private String name;
	private String surname;
	private int age;
	private Car car;
	
	public Person(String name,String surname,int age) {
		this.name = name;
		this.surname = surname;
		this.age = age;
	}
	
	public Person() {
		this.car = new Car();
	}
	
	public void setName(String name) {
		this.name = name;
	}
	public String getName() {
		return this.name;
	}
	
	public void setSurname(String surname) {
		this.surname = surname;
	}
	public String getSurname() {
		return this.surname;
	}
	
	public void setAge(int age) {
		this.age = age;
	}
	public int getAge() {
		return this.age;
	}
	
	public void setCar(Car car) {
		this.car = car;
	}
	public Car getCar() {
		if(this.car != null) {
			return this.car;
		}
		else{
			return null;
		}
	}
}
